ngraph/test/backend/auto_broadcast.in.cpp

```diff
@@ -104,74 +104,6 @@ void check_auto_bcast(
     }
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, auto_bcast_binary_elementwise)
```

Comment:
Those tests were unrelated to the operation removed in this PR. Should they really go away?

Reply:
In this test we use only opset0 operations... Also we use And operation.